src/containerapp/azext_containerapp/custom.py

```diff
@@ -1088,6 +1088,10 @@ def create_containerapps_from_compose(cmd,  # pylint: disable=R0914
     return containerapps_from_compose
 
 
+def set_workload_profile(cmd, resource_group_name, env_name, workload_profile_name, workload_profile_type=None, min_nodes=None, max_nodes=None):
```

Comment:
do we have test for the command?

Reply:
No

Reply:
Add test in `test_containerapp_env_workload_profiles_e2e_no_waits`